libs/domain/picking/src/services/picking-list-default.service.ts

```diff
@@ -54,8 +54,7 @@ export class PickingListDefaultService implements PickingListService {
     return this.adapter.startPicking(pickingList).pipe(
       catchError((e) => {
         this.upcomingPickingListId$.next(null);
-        throw e;
-        return of(null);
+        return throwError(e);
```

Comment:
This function expects a function factory of the error observable:
```suggestion
        return throwError(() => of(e));
```

Reply:
This should actually return just the error, not observable of error:
```
return throwError(() => e);
```